test/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width" />
+  <title>Tests</title>
+  <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-2.9.1.css" />
+</head>
+
+<body>
+  <div id="qunit"></div>
+  <div id="qunit-fixture"></div>
+  <script src="https://code.jquery.com/qunit/qunit-2.9.1.js"></script>
+  <script>
+    function storageMock() {
+      var storage = {};
+
+      return {
+        setItem: function (key, value) {
+          storage[key] = value || '';
+        },
+        getItem: function (key) {
+          return key in storage ? storage[key] : null;
+        },
+        removeItem: function (key) {
+          delete storage[key];
+        },
+        get length() {
+          return Object.keys(storage).length;
+        },
+        key: function (i) {
+          var keys = Object.keys(storage);
+          return keys[i] || null;
+        }
+      };
+    }
+
+    Object.defineProperty(window, 'localStorage', {
+      value: storageMock(),
+    });</script>
+
+  <script type="module" src="tests.js"></script>
+</body>
+
+</html>
```

Comment:
Add a newline at the end of each file. You can configure VSCode to do this for you automatically.